monitor/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -3,20 +3,60 @@
 <%@ taglib prefix="sec"
 	uri="http://www.springframework.org/security/tags"%>
 <%@ taglib uri="http://sargue.net/jsptags/time" prefix="time"%>
-
+<c:url value="/reload"  var ="reloadUrl"/>
+<script>
+	window.setInterval(function() {
+		$.ajax({
+			type : "GET",
+			dataType : "json",
+			url : "${reloadUrl}",
+			success : function(data) {
+				$('#ajax_reload_alert').hide()
+				data.forEach(update)
+			},
+			error : function(xhr, status, error) {
+				console.error(xhr.responseText)
+				console.error(status)
+				console.error(error.message)
+				$('#ajax_reload_alert').show()
+			}
+		});
+	}, 60000);
+	function update(data) {
+		$('#name_' + data.id).text(data.name)
+		$('#desc_' + data.id).text(data.description)
+		$('#url_' + data.id).text(data.healthUrl)
+		$('#status_' + data.id).text(
+				"App status is: " + data.lastAppTest.status)
+		$('#status_mod_' + data.id).text(data.lastAppTest.status)		
+		$('#time_' + data.id).text(parseDate(data.lastAppTest.pingTime))
+		$('#time_mod_'+ data.id).text($('#time_' + data.id).text())
+		
+	}
+	
+	function parseDate(jsonDate) {
+		const date = new Date(jsonDate.year, jsonDate.monthValue - 1,
+				jsonDate.dayOfMonth, jsonDate.hour, jsonDate.minute)
+		return moment(date).format('MMM D, YYYY h:mm A')
```

Comment:
the time should be shown in the users timezone (since moments is added now, also use that for the first loading of the time).